PurchasesTests/FoundationExtensions/NSDate+RCExtensionsTests.swift

```diff
@@ -9,8 +9,15 @@ import XCTest
 import Purchases
 
 class NSDateExtensionsTests: XCTestCase {
-    func testMillisecondsSince1970() {
+    func testMillisecondsSince1970ConvertsCorrectlyWithCurrentTime() {
         let date = NSDate()
-        expect(date.millisecondsSince1970()) == (UInt64)(date.timeIntervalSince1970 * 1000)
+        expect(date.rc_UInt64MillisecondsSince1970()) == (UInt64)(date.timeIntervalSince1970 * 1000)
```

Comment:
I realized that this test works, but only if you assume that the casting and formula is correct (i.e.: if I'd written an erroneous way of converting double to UInt64, but did it on both implementation and test, it'd still work), so I added another test case that converts from a fixed timestamp